README.md

```diff
@@ -137,6 +137,8 @@ Subsequently, the C++ tests can be ran with `./tests/nocturne_test` from within
 # Usage 
 To get a sense of available functionality in Nocturne, we have provided a few examples  in the `examples` folder of how to construct the env (`create_env.py`), how to construct particular observations (`nocturne_functions.py`), and how to render results (`rendering.py`).
 
+*Note*: by default, Nocturne will log to ```$NOCTURNE_LOG_DIR``` which is set in ```nocturne/__init__.py```. If you'd like to log somewhere else, simply change this variable.
```

Comment:
I'd say "simply change the path in the file." - since changing the NOCTURNE_LOG_DIR variable from the terminal wouldn't work